packages/cli/src/templates/expo.ts

```diff
@@ -0,0 +1,277 @@
+          await writeFile(`${cwd}/App.tsx`, APP_TSX_CONTENT, "utf8");
+          await writeFile(
+            `${cwd}/babel.config.js`,
+            BABEL_CONFIG_CONTENT,
+            "utf8",
+          );
+          await writeFile(
+            `${cwd}/metro.config.js`,
+            METRO_CONFIG_CONTENT,
+            "utf8",
+          );
+          await writeFile(
+            `${cwd}/package.json`,
+            PACKAGE_JSON_CONTENT(name),
+            "utf8",
+          );
+          await writeFile(
+            `${cwd}/PatientComponent.tsx`,
+            PATIENT_COMPONENT_CONTENT,
+            "utf8",
+          );
+          await writeFile(`${cwd}/tsconfig.json`, TSCONFIG_CONTENT, "utf8");
+
+          await writeFile(
+            `${cwd}/.prettierignore`,
+            PRETTIER_IGNORE_CONTENT,
+            "utf8",
+          );
+        },
+      },
+      {
+        title: "Add dependencies",
+        task: async ({ options: { cwd, packageManager } }) => {
+          await packageManager.install(cwd);
+          await packageManager.add(
+            cwd,
+            "@bonfhir/core",
+            "@bonfhir/query",
+            "@bonfhir/react",
+            "@bonfhir/gluestack-ui",
+            "@gluestack-style/react",
+            "@gluestack-ui/themed",
+            "expo",
+            "expo-status-bar",
+            "react",
+            "react-native",
+            "react-native-svg",
+          );
+
+          await packageManager.addDev(
+            cwd,
+            "@babel/core",
+            "@babel/plugin-transform-class-properties",
+            "@babel/plugin-transform-private-methods",
+            "@babel/plugin-transform-private-property-in-object",
+            "@types/react",
+            "@types/react-native",
+            "typescript",
+            "prettier",
+          );
+        },
+      },
+      {
+        title: "Format all files",
+        task: async ({ options: { cwd, packageManager } }) => {
+          await packageManager.runPrettier(cwd, ".");
+        },
+      },
+    ]).run({ options });
+
+    console.log();
+    console.log(
+      chalk.green(`🔥 Successfully created project ${options.name}.`),
+    );
+    console.log();
+    console.log(`To get started, run the following commands:`);
+    console.log();
+    console.log(`cd ${options.name}`);
+    console.log(`npx expo start`);
+    console.log();
+  },
+};
+
+const APP_TSX_CONTENT = `import { StatusBar } from "expo-status-bar";
+import { StyleSheet, View } from "react-native";
+import { FetchFhirClient } from "@bonfhir/core/r4b";
+import { FhirQueryProvider } from "@bonfhir/query/r4b";
+import { GluestackUIRenderer } from "@bonfhir/gluestack-ui/r4b";
+import { FhirUIProvider } from "@bonfhir/react/r4b";
+import { Text } from "@gluestack-ui/themed";
+
+import PatientComponent from "./PatientComponent";
+
+// Note: if you get [TypeError: Network request failed] errors, you may need to
+// run the following command: ipconfig getifaddr en0
```

Comment:
I think we need to reword this differently.
This command won't work on Windows, or won't work if the network device in use is not `en0`.
I could not find a reference to that specific command in the GH issue listed.
Also - it worked for me OOB on ios, so I don't know how often this error is that prevalent?
(I got a different error on Android - see global comment.)

Reply:
it didn't happen at the beginning of working on this, and now it's no longer happening for me. Not sure exactly sure why it happened to trigger the issue, but it was the first time it happened to me ever.

I will remove the comment, as it is likely uncommon and is not relevant to the project.